cmake/CMakeLists.txt

```diff
@@ -11,9 +11,14 @@ cmake_minimum_required(VERSION 3.1)
 
 include(CMakePackageConfigHelpers)
 
+set(CMAKE_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/cmake)
+
 ## Generate and install SerialboxConfig.cmake
-configure_file(${CMAKE_SOURCE_DIR}/cmake/SerialboxConfig.cmake.in
-    "${PROJECT_BINARY_DIR}/cmake/SerialboxConfig.cmake" @ONLY)
+configure_package_config_file(${CMAKE_SOURCE_DIR}/cmake/SerialboxConfig.cmake.in

```

Comment:
For setup of a config CMake file from a template, we should use `configure_package_config_file` and not `configure_file` (which is used for anything else that is not a CMake config file). See
https://cmake.org/cmake/help/v3.8/module/CMakePackageConfigHelpers.html?highlight=i